### ISSUE TITLE
ndb doesn't respect inline sourcemaps

### ISSUE THREAD

**Issue by jdalton** [GoogleChromeLabs/ndb#65]
When I load the inspector in MacOS with Node 10.7.0 it launches showing the raw source file instead of the mapped file (as the default inspector (non-ndb experience) does.

You can repro this with
```
npm install esm
touch foo.js
```
Make _foo.js_:
```js
import { log } from "console"

setTimeout(() => {
  debugger
  log("hi")
}, 2000)
```
Then
```
ndb node -r esm ./foo.js
```

<img width="503" src="https://user-images.githubusercontent.com/4303/43299830-e7bc66fa-9110-11e8-92c0-15b0d70a4532.png">

With the default experience of `node --inspect -r esm ./foo.js` it's:

<img width="324" src="https://user-images.githubusercontent.com/4303/43301160-49833902-9118-11e8-9e15-8768673d3652.png">

**Update:**

Maybe related to #57.

**Update:**

Tested 1.0.13 and it's still an issue.

**Update:**

~~Turns out I didn't have the inline source maps enabled 🤦‍♂️~~

**Comment by jdalton** [comment]
Turns out I didn't have the inline source maps enabled 🤦‍♂️

**Comment by jdalton** [comment]
Reopening as I'm still seeing an issue with source maps enabled.

**Comment by jdalton** [comment]
![](https://user-images.githubusercontent.com/4303/43624506-b5aaff82-969c-11e8-9f41-7801485a230a.gif)

**Comment by ak239** [comment]
Fixed and published as 1.0.20. Please check it out :)

**Comment by jdalton** [comment]
Works! Thank you @ak239!
